app/components/primer/alpha/action_list/heading.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render(Primer::BaseComponent.new(**@system_arguments)) do %>
+  <%= render(Primer::BaseComponent.new(tag: :h3, classes: "ActionList-sectionDivider-title", id: @section_id)) do %>
```

Comment:
@joelhawksley this heading level came up last week from an accessibility issue in Discussions. Can we add a heading level param so it can be set per implementation, with a default of `h3`?

Reply:
PR for reference: https://github.com/github/github/pull/236445/files